server/data-files/initial-tweets.json

```diff
@@ -2,23 +2,23 @@
   {
     "user": {
       "name": "Newton",
-      "avatars": "https://i.imgur.com/73hZDYK.png"
-      ,
+      "avatars": "https://i.imgur.com/73hZDYK.png",
       "handle": "@SirIsaac"
     },
     "content": {
       "text": "If I have seen further it is by standing on the shoulders of giants"
     },
-    "created_at": 1461116232227
+    "created_at": 1600205004115
   },
   {
     "user": {
       "name": "Descartes",
       "avatars": "https://i.imgur.com/nlhLi3I.png",
-      "handle": "@rd" },
+      "handle": "@rd"
+    },
     "content": {
       "text": "Je pense , donc je suis"
     },
-    "created_at": 1461113959088
+    "created_at": 1600118604115
```

Comment:
If we're dynamically updating the dates, there's no need for changes to this file.

Reply:
Added to .gitignore.

Reply:
If you revert the changes (ie. change it back to how it was... including the weird comma, then Git won't show that you've made any changes at all.

Reply:
Fixed!  Thank you!